mrmustard/lab_dev/transformations/transformations.py

```diff
@@ -211,7 +211,7 @@ def __init__(
     def representation(self) -> Bargmann:
         n_modes = len(self.modes)
         thetas = list(reshape_params(n_modes, theta=self.theta.value))[0]
-        return Bargmann(*triples.squeezing_gate_Abc(thetas))
+        return Bargmann(*triples.rotation_gate_Abc(thetas))
```

Comment:
Thanks! Since you're here can you change theta to phi? (that's how it used to be in the old Mr Mustard)

Reply:
Sure, but it is called theta also in the triplets and tests. I just want to verify that I then change every theta (related to the Rgate) to a phi to keep it consistent.